### DISCUSSION TITLE
File does not start with a RIFF ID

### DISCUSSION THREAD

**Discussion by stefan814** [rhasspy/piper#464] [Q&A]
Hello! Trying to train my own voice model but am hitting blockers in Colab - I convert the files in Audacity to .wav using mono, 16-bit, 22050hz output but still get an error when trying to extract the contents and train. Any ideas?

**Comment by synesthesiam** [comment]
That error suggests that the conversion to WAV failed somehow. If the file doesn't begin with the ASCII characters `RIFF` then it's definitely not a WAV.
